modules/storages/spec/features/storages/admin/project_storages_spec.rb

```diff
@@ -315,13 +329,23 @@
         expect(page).to have_text(project.name)
         project_storages_index_page.click_menu_item_of("Remove project", project)
 
+        # The original DeleteService would try to remove actual files from actual storages,
+        # which is of course not possible in this test since no real storage is used.
+        expect(Storages::ProjectStorages::DeleteService)
+          .to receive(:new) # rubocop:disable RSpec/MessageSpies
+          .and_wrap_original do |_original_method, *args, &_block|
+            user, model = *args.first.values
+            success_delete_service.new(user:, model:)
+          end
+
         page.within("dialog") do
           expect(page).to have_button("Remove", disabled: true)
           check "Please, confirm you understand and want to remove this file storage from this project"
-          expect(page).to have_button("Remove", disabled: false, wait: 3) # ensure button is clickable
+          wait_for(page).to have_button("Remove", disabled: false) # ensure button is clickable
           click_on "Remove"
         end
 
+        expect(page).to have_text(I18n.t(:notice_successful_delete))
```

Comment:
🟢 in a dev discussion, we settled to use strings when asserting against i18n as a better way of tracking errors in the main en translations
```suggestion
        expect(page).to have_text("Successful deletion.")
```

Reply:
Oh, that one is really good to know! Because I usually also prefer using Strings but thought the "default" here was not using them.
Thanks for mentioning it!